[FE] feat: add file navigate feature when click initial commit table item이라고 적었지만 혼합이 되버린...

pigpgw opened the pull request:
## 주요 변경사항
- tldraw에 불필요한 마우스 색상 변경 기능, 이름 입력 기능을 제거하였습니다.
- 코드리뷰 방 처음 입장시 입력한 pr 내용(파일 변경사항 리스트)을 간략하게 보여주는데 해당 아이템을 클릭시 클릭한 파일이 보이도록 하였습니다.

## 리뷰어에게...
- 훠훠...
## 관련 이슈

- closes #53

pigpgw committed:
refactor: modify init file comment

pigpgw committed:
feat: add navigate SelectedFile when click file in init file viewr

pigpgw committed:
remove: remove unnecessary feature

pigpgw committed:
Merge branch 'feat/53' of https://github.com/jungle-6-3/code-sync-fe into feat/53

pigpgw committed:
Merge branch 'main' into feat/53

pigpgw committed:
remove: remove unnecessary code

pigpgw committed:
Merge branch 'feat/53' of https://github.com/jungle-6-3/code-sync-fe into feat/53

pigpgw committed:
style: change logout button position

pigpgw committed:
style: modify logout button css style

pigpgw committed:
refactor: refactor ConversationPage bottom component create BottomGNB component

pigpgw committed:
rename: rename BottomGNB component -> PRBottomFileExplorer

pigpgw committed:
renamed: src/components/useLogoutButton.tsx -> src/components/Users/LogoutButton.tsx

loopy-lim commented on src/components/File/PRBottomFileExplorer.tsx:
```diff
@@ -0,0 +1,14 @@
+import { prInfoStore } from "@/stores/github.store";
+
+export const PRBottomFileExplorer = () => {
+  const { prInfo } = prInfoStore();
+  return (
+    <div className="p-4">
+      <h3 className="font-medium">Commit Information</h3>
+      <p className="mt-2 text-sm">
+        Branch: {prInfo.requireUserInfo.branchName} →{" "}
+        {prInfo.requestUserInfo.branchName}
+      </p>
+    </div>
+  );
+};
```
좋아요...
목요일 발표 이후에 이름에 대해서 같이 이야기 해봅시다

pigpgw replied:
넵 알겠습니다. 이름 명명을 한참 고민해도 답이 안나오네요 ㅠㅠ

loopy-lim commented on src/components/File/SelectedFileViewer.tsx:
```diff
@@ -80,7 +81,7 @@ const RemovedFile = () => {
       <Terminal className="h-4 w-4" />
       <AlertTitle className="text-2xl">Removed File</AlertTitle>
-      <AlertDescription className="text-xl">File Removed.</AlertDescription>
+      <AlertDescription className="">File Removed.</AlertDescription>
```
className을 지우셨다면 지우는 것을 추천드려요

pigpgw replied:
죄송합니다.. 바로 반영하겠습니다.

loopy-lim commented on src/components/File/SelectedFileViewer.tsx:
```diff
@@ -126,30 +127,44 @@ const InitFile = ({
-      <div className="my-2 ml-2 text-3xl font-bold">
-        Init Page : file changed status
+      <div className="my-2 ml-2 text-2xl font-bold">
+        File Changed List
```
사실 더 좋은 이름이라고 하면 File Changed를 하는 것이 좋고
이쁘게 만들고 싶다면 `font-bold`보다는 `font-semibold`를 추천드려요.

pigpgw replied:
감사합니다. 즉시 반영하도록 하겠습니다.

loopy-lim commented on src/components/File/SelectedFileViewer.tsx:
```diff
@@ -126,30 +127,44 @@ const InitFile = ({
+              <TableRow
+                key={index}
+                className=" text-4xl"
+                onClick={() => setSelectedCommitFile(file)}
```
이렇게 보니깐 정말 잘 빼두셨네요

pigpgw replied:
감사합니당

loopy-lim commented on src/pages/CreateRoomPage.tsx:
```diff
@@ -94,7 +94,9 @@ const CreateRoomPage = () => {
   return (
     <div className="relative flex h-screen flex-col items-center justify-center overflow-hidden overflow-x-hidden">
       <div className="flex w-full max-w-[30rem] flex-col gap-8">
+        <div className="m- absolute right-0 top-0 p-4">
```
`m-`와 같은 쓰지 않는 css는 제거해주세요~

pigpgw committed:
refactor: extract MainFrame component from ConversationPage

- Move complex rendering logic and state management from ConversationPage to MainFrame
- Improve code organization by separating concerns
- Reduce file complexity and enhance maintainability

loopy-lim commented on src/stores/github.store.ts:
```diff
@@ -57,7 +57,10 @@ export const prMetaDataStore = create<prMetaDataPropsStore>()((set) => ({
   },
-  setPrMetaData: (newPrMetaData) => set({ prMetaData: newPrMetaData }),
+  setPrMetaData: (newPrMetaData) => {
+    console.log(newPrMetaData);
```
`console.log`는 지워주시죠

pigpgw replied:
... 죄송합니다.

loopy-lim reviewed (changes requested):
허허... 좋습니다.... 몇가지만 다시 확인해주세요

추가적으로 클린코드에서 나오는 몇가지 규칙을 지켜주면 나중에 힘을 덜 쓰게 될 꺼에용
- 르블랑의 법칙
- 보이스카우트 법칙

pigpgw committed:
style: organize import statements order

pigpgw committed:
remove: remove unnecessary code

pigpgw committed:
remove: remove unnecessary code

loopy-lim commented on src/components/Frame/MainFrame.tsx:
```diff
@@ -0,0 +1,203 @@
+  return (
+    <ResizablePanelGroup direction="horizontal">
+      <ResizablePanel defaultSize={20}>
+        {isMessage === "folder" ? (
+          <FileTreeComponent />
+        ) : (
+          <ConversationChatting />
+        )}
```
나중에 치워주세요

loopy-lim commented on src/pages/CreateRoomPage.tsx:
```diff
@@ -94,7 +94,9 @@ const CreateRoomPage = () => {
   return (
     <div className="relative flex h-screen flex-col items-center justify-center overflow-hidden overflow-x-hidden">
       <div className="flex w-full max-w-[30rem] flex-col gap-8">
+        <div className="absolute right-0 top-0 p-4">
```
padding을 고치는 것보다 right-4, top-4가 더 좋을 수도 있어요~

loopy-lim reviewed (approved):
좋아요~ merge 하시죠
